Xamarin.Forms.Core/CornerRadiusTypeConverter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Globalization;
+
+namespace Xamarin.Forms
+{
+	[Xaml.ProvideCompiled ("Xamarin.Forms.Core.XamlC.CornerRadiusTypeConverter")]
```

Comment:
you actually do _not_ provide a compiled version of this. you don't have to do it, I will, but remove this attribute.

Reply:
Ok.